lib/medtronic/processData.js

```diff
@@ -426,6 +467,84 @@ function buildBGRecords(records) {
   return postrecords;
 }
 
+function buildCGMRecords(events) {
```

Comment:
Ah yes, that's a good idea! @pazaan , will you please review the `buildCGMRecords` function? I'll send you a binary blob containing CGM data to use until you get your BCN Link meter.